maven-release-manager/pom.xml

```diff
@@ -146,14 +148,8 @@
     </dependency>
     <dependency>
       <groupId>org.apache.maven.shared</groupId>
-      <artifactId>maven-artifact-transfer</artifactId>
-      <version>0.13.1</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.sonatype.sisu</groupId>
-          <artifactId>sisu-inject-plexus</artifactId>
-        </exclusion>
-      </exclusions>
+      <artifactId>maven-shared-utils</artifactId>
```

Comment:
why this?

Reply:
You mean the add? It was reported by `dependency:analyze` to be used.

Reply:
Remove it. Check how it is used, as it is very likely only things like StringUtils or whatever plexus-utils has... (and then drop it)

Reply:
If console color log builder, then not of course....

Reply:
Yepp, it is `import static org.apache.maven.shared.utils.logging.MessageUtils.buffer;`.